changelog/fix_fix_false_negatives_in.md

```diff
@@ -0,0 +1 @@
+* [#12430](https://github.com/rubocop/rubocop/issues/12430): Fix false negatives in `Style/RedundantLineContinuation` when continuations are inside begin nodes. ([@dvandersluis][])
```

Comment:
The changelog is a user-facing document, so replacing the term "begin node," which is an internal representation, with a different expression might make it more user-friendly :-)